pkg/lib/cockpit-components-inline-notification.jsx

```diff
@@ -90,7 +90,7 @@ InlineNotification.propTypes = {
 export const ModalError = ({ dialogError, dialogErrorDetail, id, isExpandable }) => {
     return (
         <Alert id={id} variant='danger' isInline title={dialogError} isExpandable={!!isExpandable}>
-            { dialogErrorDetail && <p> {dialogErrorDetail} </p> }
+            { typeof dialogErrorDetail === 'string' ? <p>{dialogErrorDetail}</p> : dialogErrorDetail }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19520-20231024-085000-668e64da-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)